composer.json

```diff
@@ -195,6 +200,12 @@
             "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php"
         ]
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/atomiix/jwt"
```

Comment:
👀

Reply:
We'll probably have to move this repo to the PrestaShop org. It's a fork of `lcobucci/jwt` that brings php 7.2 to 8.1 compatibility.